polyfills/Element/prototype/matches/config.json

```diff
@@ -10,6 +10,7 @@
 	],
 	"browsers": {
 		"ie": "6 - *",
+		"ie_mob": "10 - *",
```

Comment:
Sure... In any case, I think I still need to understand how pull requests works, as that was supposed to be ANOTHER pull request for another issue, but anyways..
![wp_ss_20170126_0001](https://cloud.githubusercontent.com/assets/201414/22336784/28319e60-e3c2-11e6-8731-0f509f55f553.jpg)
![wp_ss_20170126_0002](https://cloud.githubusercontent.com/assets/201414/22336785/28cc3268-e3c2-11e6-9b7c-d8bb46c614a3.jpg)